options/locale/locale_en-US.ini

```diff
@@ -3530,6 +3530,7 @@ runs.actors_no_select = All actors
 runs.status_no_select = All status
 runs.no_results = No results matched.
 runs.no_runs = The workflow has no runs yet.
+runs.no_commit_message = empty commit message
```

Comment:
Empty commits can appear in other contexts as well, I would make it top-level `no_commit_message` or `empty_commit_message`.

Reply:
done.